website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4092,13 +4271,22 @@ that are likely associated with that transaction (both the user and product matc
 "natural join" - one in which the same field names are used on both the left-hand and right-hand sides of the join -
 and is specified with the `using` keyword:
```

Comment:
Should this paragraph be in a language-java block? Similarly for the other Java-only schema transforms - it seems theres a lot of language that needs to be hidden when Python/Go are selected.

Reply:
If you'd rather not open that can of worms (there are a lot of schema transforms...), we could also just edit the Select/GroupBy-relevant sections in this PR, and leave the other ones as-is for now (and address in a follow-up PR).

Reply:
I think we should only edit the Select/GroupBy sections in this PR and create another PR, it's getting difficult to keep track of all the sections

Reply:
Agreed